src/IO/ConnectionTimeouts.h

```diff
@@ -53,7 +64,10 @@ struct ConnectionTimeouts
           receive_timeout(receive_timeout_),
           tcp_keep_alive_timeout(tcp_keep_alive_timeout_),
           http_keep_alive_timeout(http_keep_alive_timeout_),
-          secure_connection_timeout(connection_timeout)
+          secure_connection_timeout(connection_timeout),
```

Comment:
I suppose, we should use default `receive_timeout` value for new receive timeouts?